apis/v1beta1/validation/httproute.go

```diff
@@ -334,6 +341,30 @@ func validateHTTPHeaderModifier(filter gatewayv1b1.HTTPHeaderFilter, path *field
 	return errs
 }
 
+func validateHTTPRouteTimeouts(timeouts *gatewayv1b1.HTTPRouteTimeouts, path *field.Path) field.ErrorList {
+	var errs field.ErrorList
+	errorString := fmt.Sprintf("timeout value must be greater than or equal to %d ms", timeoutMinValue)
+	if timeouts.BackendRequest != nil {
+		backendTimeout := int(timeouts.BackendRequest.Duration.Milliseconds())
+		if backendTimeout < timeoutMinValue && backendTimeout != 0 {
+			errs = append(errs, field.Invalid(path.Child("backendRequest"), timeouts.BackendRequest.Duration, errorString))
+		}
+		if timeouts.Request != nil {
+			if backendTimeout > int(timeouts.Request.Duration.Milliseconds()) {
```

Comment:
Sorry, one more I missed
```suggestion
			timeout := int(timeouts.Request.Duration.Milliseconds())
			if backendTimeout > timeout && timeout != 0 {
```

Reply:
addressed in 2a3078bc8351b2d4b990811700971d2066694c62